pupil_src/shared_modules/video_capture/tests/utils/test_utils.py

```diff
@@ -0,0 +1,47 @@
+import os
+from video_capture.utils import RenameSet
+
+
+def test_rename_file(tmpdir):
+    tmp_dir = tmpdir.mkdir("rename_data")
+    file_lst = [
+        # Audio file
+        "audio_00010000.mp4", "audio_00010000.time",
+        # ID0 file
+        "Pupil Cam1 ID0.mp4", "Pupil Cam1 ID0.time",
+        "Pupil Cam1 ID0_001.mp4", "Pupil Cam1 ID0_001.time",
+        # ID2 file
+        "Pupil Cam1 ID2.mp4", "Pupil Cam1 ID2.time",
+        "Pupil Cam1 ID2_001.mp4", "Pupil Cam1 ID2_001.time",
+        ]
+    for fn in file_lst:
+        f = tmp_dir.join(fn)
+        f.write("foobar"*2)
+    match_pattern = "*.time"
+    rename_set = RenameSet(tmp_dir, match_pattern)
+    # Rename to eye0
+    rename_set.rename("Pupil Cam(0|1) ID0", "eye0")
+    assert all(
+        elem in os.listdir(tmp_dir) for elem in
+        ["eye0.mp4", "eye0_001.mp4"])
+    assert not any(
+        elem in os.listdir(tmp_dir) for elem in
+        ["Pupil Cam1 ID0.mp4", "Pupil Cam1 ID0_001.mp4"])
+    # Rename to world
+    rename_set.rename("Pupil Cam(0|1) ID2", "world")
```

Comment:
Is this kind of RE compatible with `glob`?

Reply:
No, glob use fnmatch which only support Unix shell-style wildcards ([implement details](https://github.com/python/cpython/blob/master/Lib/fnmatch.py#L74))